### DISCUSSION TITLE
How to adjust column size ?

### DISCUSSION THREAD

**Discussion by AliNB0** [DirectoryLister/DirectoryLister#1187] [Help & Support]
how to adjust column size of the file name, size and date ?
![2](https://github.com/DirectoryLister/DirectoryLister/assets/116026049/49ab928e-f001-4fc5-a2bd-5ef064ce19b0)

**Comment by jult** [comment]
Yes! And in addition options to make it more compact. The padding is really way too high. Lines can be stacked together, which would look much better i.m.o. Note how this too is always more compact in practically ALL other file list indexers out there. I have no idea why anyone would want such thick padding per line. Why not do a full width of browser screen (so not the weird center block with a set width that is shown different for each user)? And why not list the folders and files all the way to the left with a normal margin, just like any other directory lister would? I don't get these strange design ideas.  Who even truly prefers to see a listing this way over the normal standard way? And purple, really?
